doc/nrf/templates/application_README.rst

```diff
@@ -0,0 +1,272 @@
+
+The application provides predefined configuration files for typical use cases.
+You can find the configuration files in the :file:`XXX` directory.
+
+The following files are available:
+
+* :file:`filename.conf` - Specific scenario.
+  This configuration file ...
+
+Building and running
+********************
+
+.. note::
+   * Include the standard text for building - either ``.. include:: /includes/application_build_and_run.txt`` or ``.. include:: /includes/application_build_and_run_ns.txt`` for the build targets that use :ref:`Cortex-M Security Extensions <app_boards_spe_nspe>`.
+   * The main supported IDE for |NCS| is |VSC|, with the |nRFVSC| installed.
+     Therefore, build instructions for the |nRFVSC| are required.
+     Build instructions for the command line are optional.
+   * See the link to the `nRF Connect for Visual Studio Code`_ documentation site for basic instructions on building with the extension.
+   * If the application uses a non-standard setup, point it out and link to more information, if possible.
+
+.. |application path| replace:: :file:`applications/XXX`
+
+.. include:: /includes/application_build_and_run.txt
+
+Some title*
+===========
+
+.. note::
+   If required, add subsections for additional build instructions.
+   Use these subsections sparingly and only if the content does not fit into other sections (mainly `Configuration*`_).
+   If the additional build instructions are valid for other applications as well, consider adding them to the :ref:`getting_started` section instead and linking to them.
+
+Testing
+=======
+
+|test_application|
+
+#. |connect_kit|
+#. |connect_terminal|
+#. Reset the development kit.
+#. And so on ...
+
+.. note::
+   * Use the shortcuts provided in :file:`doc/nrf/shortcuts.txt` to keep the wording consistent.
+   * If there are different ways of testing, introduce them in this section (for example, "After programming the application to your development kit, you can test it either by ...") and add subsections for the different scenarios.
+
+Application output*
+===================
+
+.. note::
+   Add the full output of the application in this section or include parts of the output in the testing steps in the previous section.
+
+The application shows the following output:
+
+.. code-block:: console
+
+   [00:00:02.029,174] <inf> zigbee_app_utils: Zigbee stack initialized
+
+References*
+***********
+
+.. note::
+   Provide a link to other relevant documentation for the user to get more information.
+
+.. tip::
+   Do not include links to documents that are common to all or many of our applications. For example, :ref:`getting_started` section.
+
+Implementation in other applications and samples*
+=================================================
+
+.. note::
+   * Add details about the projects that demonstrate or implement the application.
+   * Add details about the samples that show aspects of the application.
```

Comment:
```suggestion
References*
***********

.. note::
   Provide a link to other relevant documentation for the user to get more information.

.. tip::
   Do not include links to documents that are common to all or many of our applications. For example, :ref:`getting_started` section.

Related projects and applications*
==================================

.. note::
   * Add links to projects and/or applications that demonstrate or implement some or all of the features of this application. For example, the Asset Tracker v2 application is part of the Asset Tracker project. Matter Weather Station application is part of the Matter project, and so on.


Related samples*
=========================

.. note::
   * Add links to the samples that show aspects of the application. A sample showcases one feature or component. An application uses several features or components. Include links to samples that showcase features included in the application.
```

Reply:
@umapraseeda , @divipillai @b-gent : Your thoughts on this change?

Reply:
Regarding : "For example, the Asset Tracker v2 application is part of the Asset Tracker project." - As per my understanding both are different. Both use nRF9160 DK or Thingy:91 but the implementation is different.
Asset tracker project : https://nordicsemiconductor.github.io/asset-tracker-cloud-docs/v2.1.x/docs/project/Index.html
Asset tracker application : https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/applications/asset_tracker_v2/README.html
I think Its would be better to remove this sentence.

Reply:
The sentence about Asset Tracker can go. What is crucial in this edit is the structure of the sections, which matches the original proposal from Joel.